src/components/content-tab-overview.tsx

```diff
@@ -234,15 +237,43 @@ export function ContentTabOverview( { selectedSite }: ContentTabOverviewProps )
 					className={ cx(
 						'w-full min-h-40 max-h-60 overflow-hidden rounded-sm border border-a8c-gray-5 bg-a8c-gray-0 mb-2 flex items-center justify-center',
 						loading && skeletonBg,
-						isThumbnailError && 'border-none'
+						isThumbnailError && 'border-none',
+						siteRunning && 'hover:border-a8c-blueberry duration-300'
 					) }
 				>
 					{ isThumbnailError && ! loading && (
 						<div className="flex items-center justify-center w-full h-full leading-5 text-a8c-gray-50">
 							{ __( 'Preview unavailable' ) }
 						</div>
 					) }
-					{ ! loading && (
+					{ ! loading && siteRunning && (
+						<button
+							aria-label={ __( 'Open WP Admin' ) }
+							className={ cx( `relative group` ) }
+							onClick={ () => getIpcApi().openSiteURL( selectedSite.id, '/wp-admin' ) }
+						>
+							<div
+								className={ cx(
+									`opacity-0 group-hover:opacity-90 group-hover:bg-white duration-300 absolute size-full flex justify-center items-center bg-white text-a8c-blueberry`
+								) }
+							>
+								{ __( 'Open Site' ) }
```

Comment:
```suggestion
								{ __( 'Open site' ) }
```

This seems to be the spelling according to the design

Reply:
I fixed the letter casing, thanks.